### DISCUSSION TITLE
Is there a way to prevent users from creating global activities?

### DISCUSSION THREAD

**Discussion by matheusjardimb** [kimai/kimai#3470] [Questions & Answers]
Hello there,
I'm new to Kimai and I'm setting up a new server to handle time tracking for multiple projects and I can't find a way to solve this:

I need users to be able to only create activities related to the projects they are into, preventing the creation of global activities (for certain users).

Is there a way of achieving this?

**Comment by kevinpapst** [comment]
I am not entirely sure honestly. My first guess is: no. 
But you can try to achieve something with team permissions and then assign the "team member create activity" permission.
